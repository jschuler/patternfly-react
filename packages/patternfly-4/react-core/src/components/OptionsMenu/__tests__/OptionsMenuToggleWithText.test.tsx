// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { OptionsMenuToggleWithText } from '../OptionsMenuToggleWithText';

describe('OptionsMenuToggleWithText test', () => {
  it('OptionsMenuToggleWithText should match snapshot', () => {
    const view = mount(
      <OptionsMenuToggleWithText
        parentId={"''"}
				toggleText={<>ReactNode</>}
				toggleTextClassName={"''"}
				toggleButtonContents={undefined}
				toggleButtonContentsClassName={"''"}
				onToggle={() => null as any}
				onEnter={undefined}
				isOpen={false}
				isPlain={false}
				isFocused={false}
				isHovered={false}
				isActive={false}
				isDisabled={false}
				parentRef={undefined}
				ariaHasPopup={undefined}
				aria-label={"'Options menu'"}
      />);
    expect(view).toMatchSnapshot();
  });
});
