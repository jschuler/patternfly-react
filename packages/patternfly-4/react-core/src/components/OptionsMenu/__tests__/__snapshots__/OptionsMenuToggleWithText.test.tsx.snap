// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionsMenuToggleWithText test OptionsMenuToggleWithText should match snapshot 1`] = `
<Component
  aria-label="'Options menu'"
  isActive={false}
  isDisabled={false}
  isFocused={false}
  isHovered={false}
  isOpen={false}
  isPlain={false}
  onToggle={[Function]}
  parentId="''"
  toggleButtonContentsClassName="''"
  toggleText={
    <React.Fragment>
      ReactNode
    </React.Fragment>
  }
  toggleTextClassName="''"
>
  <div
    className="pf-c-options-menu__toggle pf-m-text"
  >
    <span
      className="pf-c-options-menu__toggle-text ''"
    >
      ReactNode
    </span>
    <button
      aria-expanded={false}
      aria-haspopup="listbox"
      aria-label="'Options menu'"
      className="pf-c-options-menu__toggle-button ''"
      id="''-toggle"
      onClick={[Function]}
    />
  </div>
</Component>
`;
